Automattic/jetpack
# Protect: Hide Scan Report When No Data

## Proposed changes:

* Ensures the scan data is either available or in progress prior to rendering the scan report on the home page in the Protect plugin.

### Other information:


## Jetpack product discussion



N/A

## Does this pull request change what data or activity we track or use?



No

## Testing instructions:






* Starting with a new site, connect for free and ensure the scan report doesn't render until it has the theme and plugin data available from a completed or in-progress scan.
